app/views/users/new.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <center>
   <section>
-    <%= form_with model: @user, url: "/users", method: :post do |f| %>
+    <%= form_with model: @user, url: "/users", method: :post, local: true do |f| %>
       <%= f.label :name %>
```

Comment:
Good catch, this created a bizarre error in the development server that isn't caught by the tests.